provisioning/baremetal_pod.go

```diff
@@ -325,6 +326,12 @@ func createContainerMetal3BaremetalOperator(images *Images, config *metal3iov1al
 				Value: metal3AuthRootDir,
 			},
 		},
+		Resources: corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("100m"),
+				corev1.ResourceMemory: resource.MustParse("300Mi"),
```

Comment:
I read through @markmc's link (https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#resources-and-limits), and I'm wondering what "Specifying requests without limits allows us to ensure that all components receive their required minimum resources and that they are able to burst to use more resources as needed." means for us.

300Mi might be reasonable as the minimum in steady-state, but we know we need to burst higher whenever Ironic does anything with images. Do we need to include that in our minimum consideration?

Reply:
Let me go back and actually run the prometheus query while it's provisioning some workers.

Reply:
Since these are values we may need to tune over time, it would be good to include in the commit message (or a comment in the code?) how the current values were derived (what query, what load, etc.) so that can either be reproduced or tweaked the next time.

Reply:
Quoting from the doc "The memory request of cluster components should be set to a value 10% higher than their 90th percentile actual consumption over a standard end-to-end suite run." So if we took 300Mi as the steady-state value, then maybe our request here should be 330Mi?